recipes/profimp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "profimp" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "899579c0076c19c8532c00be624a2e7b8ebc27ff97bec28c8b3759623ec84518" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - profimp = profimp.main:main
+
+requirements:
+  host:
+    - python
+    - pbr
+    - pip
+  run:
+    - python
+
+test:
+  commands:
```

Comment:
Should this also have import tests?

Reply:
Looks like it is mostly a CLI tool:

https://github.com/boris-42/profimp#usage

Reply:
It is mostly a CLI, and the CLI performs the imports